keras/metrics.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+from . import backend as K
+
+
+def accuracy(y_pred, y_true, class_mode):

```

Comment:
If we are going to use `__name__` as the display name of a metric, then this should be renamed "acc" to match expectations in the existing codebase. But another way may be possible. How about some polymorphisme: elements in the metrics array can be either functions, strings (an alias to a default function), or tuples. When tuple, it is expected that the first element is a string (the display name of the metric) and the second is a function or a string alias of a function.